.env.example

```
@@ -1,369 +1,116 @@
+# DevSkyy Platform Environment Configuration
+# Copy this file to .env and fill in your values
 
-# ============================================================================
-# ENVIRONMENT SETTINGS
-# ============================================================================
+# =============================================================================
+# Application
+# =============================================================================
 ENVIRONMENT=development
-DEBUG=false
+DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBUG key should go before the ENVIRONMENT key

(UnorderedKey)